core/common/src/main/java/alluxio/security/login/LoginModuleConfiguration.java

```diff
@@ -64,6 +64,26 @@
    */
   public LoginModuleConfiguration() {}
 
+  /**
+   * Gets and returns configuration entries for the specified application.
+   * <p>
+   * Verifies whether the lowercase name of the application corresponds
+   * to either {@link AuthType#SIMPLE} or {@link AuthType#CUSTOM}. In
+   * which case returns {@link LoginModuleConfiguration#SIMPLE}.
```

Comment:
I didn't understand this sentence: " In which case returns {@link LoginModuleConfiguration#SIMPLE}"

Reply:
Fixed.